ui/lib/config-ui/addon/routes/login-settings/rule/details.js

```diff
@@ -4,5 +4,26 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
 
-export default class LoginSettingsRuleDetailsRoute extends Route {}
+export default class LoginSettingsRuleDetailsRoute extends Route {
+  @service api;
+  @service store;
+
+  async model() {
+    const { name } = this.paramsFor('login-settings.rule');
+
+    const adapter = this.store.adapterFor('application');
+
+    const rule = await adapter.ajax(`/v1/sys/config/ui/login/default-auth/${encodeURI(name)}`, 'GET');
+
+    return { rule };
+  }
```

Comment:
This is definitely a fine way to do this! But we do want to make sure that we handle error scenarios if the request fails. We can do this a couple ways, by wrapping this in a try..catch or leveraging the error templates already in place (I think 😅 ), by directly returning the request. 

Then the template will just have to be updated from passing `this.model.rule` to just `this.model`
```suggestion
  model() {
    const { name } = this.paramsFor('login-settings.rule');

    const adapter = this.store.adapterFor('application');

   return adapter.ajax(`/v1/sys/config/ui/login/default-auth/${encodeURI(name)}`, 'GET');
  }
```